specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+      "description": "Project onboarding info.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets name of the AzureDevOps Project.",
+          "type": "string"
+        },
+        "autoDiscovery": {
+          "$ref": "#/definitions/AutoDiscovery"
+        },
+        "repos": {
+          "description": "Gets or sets repositories.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "AzureDevOpsProjectProperties": {
+      "description": "AzureDevOps Project properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "projectId": {
+          "description": "Gets or sets AzureDevOps Project Id.",
+          "type": "string"
+        },
+        "orgName": {
+          "description": "Gets or sets AzureDevOps Org Name.",
+          "type": "string"
+        },
+        "autoDiscovery": {
+          "$ref": "#/definitions/AutoDiscovery"
+        }
+      }
+    },
+    "AzureDevOpsRepo": {
+      "description": "Azure DevOps Repo Proxy Resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AzureDevOpsRepoProperties"
+        }
+      }
+    },
+    "AzureDevOpsRepoListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureDevOpsRepo"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureDevOpsRepoProperties": {
+      "description": "AzureDevOps Repo properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "repoId": {
+          "description": "Gets or sets Azure DevOps repo id.",
+          "type": "string"
+        },
+        "repoUrl": {
+          "description": "Gets or sets AzureDevOps repo url.",
+          "type": "string"
+        },
+        "orgName": {
+          "description": "Gets or sets AzureDevOps Org Name.",
+          "type": "string"
+        },
+        "projectName": {
+          "description": "Gets or sets AzureDevOps Project Name.",
+          "type": "string"
+        },
+        "visibility": {
+          "description": "Gets or sets AzureDevOps repo visibility, whether it is public or private etc.",
+          "type": "string"
+        },
```

Comment:
Is it intended for one or both of these to be supplied in the create request? Do you intend for either of these properties to be marked as "required" here?

Reply:
all of these except for `ActionableRemediation` are readonly.